Topic 0/src/proxy/ProxyDatabase.java

```diff
@@ -0,0 +1,12 @@
+package proxy;
+
+public class ProxyDatabase {
+	Database db;
+	
+	public void connect() {
```

Comment:
but you are loosing the reference to you database, you only have a connect method and nothing else.
I would like to see a getData method or something like that.